src/random_test.cpp

```diff
@@ -1607,20 +1608,64 @@ void Table::DeleteRandomRow(Thd1 *thd) {
       }
     }
   }
-
   std::string sql = "DELETE FROM " + name_ + " WHERE " +
-                    columns_->at(where)->name_ + "=" +
-                    columns_->at(where)->rand_value();
+                   columns_->at(where)->name_;
+  if (prob <= 90)
+    sql += " = " + columns_->at(where)->rand_value();
```

Comment:
replace auto where = rand_int(columns_->size() - 1);
auto where = columns_->at( rand_int(columns_->size() - 1))

Reply:
These changes are not part of the PR, but I have fixed it.